asset/css/main.css

```diff
@@ -6,37 +6,30 @@
   --text: #000;
```

Comment:
Is this file the output of the LESS compiling?

Reply:
Yes, but as I can see un the comment below, let's remove the folder and file 💃